sdk/storage/azure-storage-file-share/out/azure/storage/fileshare/_file_client.pyi

```diff
@@ -0,0 +1,51 @@
+    @classmethod
+    def from_file_url(cls: Any, file_url: str, snapshot: Optional[Union[str, Dict[str, Any]]]=..., credential: Optional[Any]=..., **kwargs: Any) -> ShareFileClient: ...
+    @classmethod
+    def from_connection_string(cls: Any, conn_str: str, share_name: str, file_path: str, snapshot: Optional[Union[str, Dict[str, Any]]]=..., credential: Optional[Any]=..., **kwargs: Any) -> ShareFileClient: ...
+    def acquire_lease(self, lease_id: Optional[str]=..., **kwargs: Any) -> BlobLeaseClient: ...
+    def create_file(self, size: int, file_attributes: Union[str, NTFSAttributes]=..., file_creation_time: Union[str, datetime]=..., file_last_write_time: Union[str, datetime]=..., file_permission: Optional[str]=..., permission_key: Optional[str]=..., **kwargs: Any) -> Dict[str, Any]: ...
+    def upload_file(self, data: Any, length: Optional[int]=..., file_attributes: Union[str, NTFSAttributes]=..., file_creation_time: Union[str, datetime]=..., file_last_write_time: Union[str, datetime]=..., file_permission: Optional[str]=..., permission_key: Optional[str]=..., **kwargs: Any) -> Dict[str, Any]: ...
+    def start_copy_from_url(self, source_url: str, file_permission_copy_mode: Optional[str]=..., file_permission: Optional[str]=..., file_permission_key: Optional[str]=..., copy_file_smb_info: Any=..., **kwargs: Any) -> Any: ...
+    def abort_copy(self, copy_id: Union[str, FileProperties], **kwargs: Any) -> None: ...
+    def download_file(self, offset: Optional[int]=..., length: Optional[int]=..., **kwargs: Any) -> Iterable[bytes]: ...
+    def delete_file(self, **kwargs: Any) -> None: ...
+    def get_file_properties(self, **kwargs: Any) -> FileProperties: ...
+    def set_http_headers(self, content_settings: ContentSettings, file_attributes: Union[str, NTFSAttributes]=..., file_creation_time: Union[str, datetime]=..., file_last_write_time: Union[str, datetime]=..., file_permission: Optional[str]=..., permission_key: Optional[str]=..., **kwargs: Any) -> Dict[str, Any]: ...
+    def set_file_metadata(self, metadata: Optional[Dict[str, Any]]=..., **kwargs: Any) -> Dict[str, Any]: ...
+    def upload_range(self, data: bytes, offset: int, length: int, **kwargs: Any) -> Dict[str, Any]: ...
+    def upload_range_from_url(self, source_url: str, offset: int, length: int, source_offset: int, **kwargs: Any) -> Dict[str, Any]: ...
+    def get_ranges(self, offset: Optional[int]=..., length: Optional[int]=..., **kwargs: Any) -> List[Dict[str, int]]: ...
+    def clear_range(self, offset: int, length: int, **kwargs: Any) -> Dict[str, Any]: ...
+    def resize_file(self, size: int, **kwargs: Any) -> Dict[str, Any]: ...
+    def list_handles(self, **kwargs: Any) -> ItemPaged[Handle]: ...
+    def close_handle(self, handle: Union[str, HandleItem], **kwargs: Any) -> Dict[str, int]: ...
```

Comment:
For the close handles response, suggest renaming the failure count to `close_failure_count`

Reply:
Same comment for DirectoryClient

Reply:
failed_handles_count is what we decided with One SDK when release fileshare package. All SDKs are using this name

Reply:
Actually all the SDKs were using different names :(
So at this point we're unable to align with the other SDKs. However in this case it's more about using a name to describe the behavior, which in this case it's number of handles that failed to close. Thoughts?

Reply:
in .NET it's also FailedHandlesCount https://github.com/Azure/azure-sdk-for-net/blob/feature/storage/stg71-base/sdk/storage/Azure.Storage.Files.Shares/src/Models/CloseHandlesResult.cs

Checking java now

Reply:
Synced with other SDKs I will make the change!

Reply:
ah it seems I got the wrong info..